pyproject.toml

```diff
@@ -95,7 +96,7 @@ opentelemetry = ["opentelemetry-instrumentation-asgi"]
 piccolo = ["piccolo"]
 picologging = ["picologging"]
 prometheus = ["prometheus-client"]
-pydantic = ["pydantic", "email-validator", "pydantic-extra-types"]
+pydantic = ["pydantic<2.10", "email-validator", "pydantic-extra-types"]
```

Comment:
you can relax that since you fixed it elsewhere

Reply:
Working on it at least 😅